manifests/0000_30_openshift-apiserver-operator_07_deployment.yaml

```
@@ -42,7 +42,7 @@ spec:
           name: serving-cert
         env:
         - name: IMAGE
-          value: quay.io/openshift/origin-hypershift:v4.0
+          value: quay.io/openshift/origin-openshift-apiserver:v4.0
```

```suggestion
          value: quay.io/openshift/origin-openshift-apiserver:v4.2
```
I don't believe this images will be tagged with anything before v4.2